ignition.repos no longer valid

Is it no longer getting used anywhere?
```
Cloning into 'src/dsim_repos_index'...
remote: Enumerating objects: 101, done.
remote: Counting objects: 100% (101/101), done.
remote: Compressing objects: 100% (61/61), done.
remote: Total 537 (delta 54), reused 65 (delta 32), pack-reused 436
Receiving objects: 100% (537/537), 146.89 KiB | 1.93 MiB/s, done.
Resolving deltas: 100% (303/303), done.
EEEEEEEEEE
=== ./ign_cmake1 (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-cmake.git': abort: HTTP Error 404: Not Found
=== ./ign_cmake2 (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-cmake.git': abort: HTTP Error 404: Not Found
=== ./ign_common (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-common.git': abort: HTTP Error 404: Not Found
=== ./ign_gui (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-gui.git': abort: HTTP Error 404: Not Found
=== ./ign_math (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-math.git': abort: HTTP Error 404: Not Found
=== ./ign_msgs (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-msgs.git': abort: HTTP Error 404: Not Found
=== ./ign_plugin (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-plugin.git': abort: HTTP Error 404: Not Found
=== ./ign_rendering (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-rendering.git': abort: HTTP Error 404: Not Found
=== ./ign_tools (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-tools.git': abort: HTTP Error 404: Not Found
=== ./ign_transport (hg) ===
Could not clone repository 'https://bitbucket.org/ignitionrobotics/ign-transport.git': abort: HTTP Error 404: Not Found
make: *** [Makefile:70: update_sources] Error 1
```

--- a/ignition.repos
+++ b/ignition.repos
@@ -1,11 +1,11 @@
 repositories:
-  ign_cmake1      : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-cmake.git',         version: 'ign-cmake1' }
-  ign_cmake2      : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-cmake.git',         version: 'ign-cmake2' }
-  ign_tools       : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-tools.git',         version: 'ign-tools0' }
-  ign_math        : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-math.git',          version: 'ign-math6' }
-  ign_common      : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-common.git',        version: 'ign-common3' }
-  ign_msgs        : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-msgs.git',          version: 'ign-msgs2' }
-  ign_transport   : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-transport.git',     version: 'ign-transport5' }
-  ign_rendering   : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-rendering.git',     version: 'ign-rendering2' }
-  ign_gui         : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-gui.git',           version: 'ign-gui0' }
-  ign_plugin      : { type: 'hg',  url: 'https://bitbucket.org/ignitionrobotics/ign-plugin.git',        version: 'ign-plugin1' }
+  ign_cmake1      : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-cmake.git',         version: 'ign-cmake1' }
+  ign_cmake2      : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-cmake.git',         version: 'ign-cmake2' }
+  ign_tools       : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-tools.git',         version: 'ign-tools0' }
+  ign_math        : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-math.git',          version: 'ign-math6' }
+  ign_common      : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-common.git',        version: 'ign-common3' }
+  ign_msgs        : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-msgs.git',          version: 'ign-msgs2' }
+  ign_transport   : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-transport.git',     version: 'ign-transport5' }
+  ign_rendering   : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-rendering.git',     version: 'ign-rendering2' }
+  ign_gui         : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-gui.git',           version: 'ign-gui0' }
+  ign_plugin      : { type: 'git',  url: 'https://github.com/ignitionrobotics/ign-plugin.git',        version: 'ign-plugin1' }

